build-support/scripts/devtools.sh

```diff
@@ -165,10 +165,10 @@ function install_unversioned_tool {
     local install="$2"
```

Comment:
The changes in here are related to a weird behaviour when running in CI. Circle-CI seems to fail setting the bash colonization options even if the env variable for colors print is set to 1. see [this](https://app.circleci.com/pipelines/github/hashicorp/consul/32228/workflows/cdfacfa6-5309-416f-a341-1363f3daec6f/jobs/698057)

Reply:
@dhiaayachi : it looks like this branch now has merge conflicts with this file?

Reply:
@jkirschner-hashicorp it should be fixed now